--- src/utils/cas/configfile.c.orig
+++ src/utils/cas/configfile.c
@@ -44,16 +44,16 @@
 		"# font_size - size the font\n",
 		"# source_image - image where the text will be writen\n",
 		"# *_line - x,y,[1/0] enabled or disabled\n\n",
-		"font /usr/share/fonts/corefonts/times.ttf\n",
+		"font /usr/share/fonts/truetype/ttf-bitstream-vera/Vera.ttf\n",
 		"font_size 10.5\n",
-		"source_image /usr/share/pixmaps/stat.png\n",
+		"source_image /usr/share/cas/stat.png\n",
 		"first_line 23,19,1\n",
 		"second_line 23,36,1\n",
 		"third_line 23,54,1\n",
 		"fourth_line 23,72,1\n",
 		"fifth_line 23,89,1\n",
 		"sixth_line 23,106,1\n",
-		"template /usr/share/pixmaps/tmp.html\n"
+		"template /usr/share/cas/tmp.html\n"
 	};
 
 	path = get_path("casrc");
