--- configure.in.orig
+++ configure.in
@@ -259,7 +259,7 @@
 AM_PATH_GDLIBCONFIG([2.0.0],
 [
 	CAS_DEFS="-D__GD__"
-	GDLIB_LIBS="-lgd $GDLIB_LIBS"
+	GDLIB_LIBS="-lgd"
 ], [
     AC_MSG_NOTICE([
 	WARNING: gd-lib >= 2.0.0, or a functional gd.h (from gd-devel) not found.
--- configure.orig
+++ configure
@@ -6336,7 +6336,7 @@
         GDLIB_LIBS="`$GDLIB_CONFIG_WITH_ARGS --libs`"
 
 	CAS_DEFS="-D__GD__"
-	GDLIB_LIBS="-lgd $GDLIB_LIBS"
+	GDLIB_LIBS="-lgd"
 
       else
         GDLIB_VERSION=
