--- a/src/ED2KLinkParser.cpp
+++ b/src/ED2KLinkParser.cpp
@@ -32,6 +32,7 @@
 #include <sstream>
 #include <iostream>
 #include <fstream>
+#include <cstdlib>
 
 #ifdef __APPLE__
 	#include <CoreServices/CoreServices.h>
--- a/src/ExternalConnector.h
+++ b/src/ExternalConnector.h
@@ -143,7 +143,7 @@
 	bool Parse_Command(const wxString& buffer);
 	void GetCommand(const wxString &prompt, char* buffer, size_t buffer_size);
 	const CECPacket *SendRecvMsg_v2(const CECPacket *request) { return m_ECClient->SendRecvPacket(request); }
-	void ConnectAndRun(const wxString &ProgName, const wxString& ProgName);
+	void ConnectAndRun(const wxString &ProgName, const wxString& ProgVersion);
 	void ShowGreet();
 
 	//
