Description: Handle libupnp3->libupnp6 transition
Author: Nick Leverton <nick@leverton.org>
Forwarded: yes
Bug: http://forum.amule.org/index.php?topic=19738
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=670962
Index: amule/src/UPnPBase.cpp
===================================================================
--- amule.orig/src/UPnPBase.cpp	2011-04-12 00:27:41.647976495 +0200
+++ amule/src/UPnPBase.cpp	2012-05-03 19:44:43.141922709 +0200
@@ -1299,7 +1299,7 @@
 		} else {
 #if 0
 			TvCtrlPointHandleSubscribeUpdate(
-				es_event->PublisherUrl,
+				UpnpString_get_String(es_event->PublisherUrl),
 				es_event->Sid,
 				es_event->TimeOut );
 #endif
@@ -1324,7 +1324,7 @@
 		int TimeOut = 1801;
 		int ret = UpnpSubscribe(
 			upnpCP->m_UPnPClientHandle,
-			es_event->PublisherUrl,
+			UpnpString_get_String(es_event->PublisherUrl),
 			&TimeOut,
 			newSID);
 		if (ret != UPNP_E_SUCCESS) {
@@ -1333,13 +1333,13 @@
 				msg.str(), es_event->ErrCode, NULL, NULL);
 		} else {
 			ServiceMap::iterator it =
-				upnpCP->m_ServiceMap.find(es_event->PublisherUrl);
+				upnpCP->m_ServiceMap.find(UpnpString_get_String(es_event->PublisherUrl));
 			if (it != upnpCP->m_ServiceMap.end()) {
 				CUPnPService &service = *(it->second);
 				service.SetTimeout(TimeOut);
 				service.SetSID(newSID);
 				msg2 << "Re-subscribed to EventURL '" <<
-					es_event->PublisherUrl <<
+					UpnpString_get_String(es_event->PublisherUrl) <<
 					"' with SID == '" <<
 					newSID << "'.";
 				AddDebugLogLineC(logUPnP, msg2);
