diff -ur amule-1.2.6+rc8.orig/src/ListenSocket.cpp amule-1.2.6+rc8/src/ListenSocket.cpp
--- amule-1.2.6+rc8/src/ListenSocket.cpp	2004-12-22 01:20:09.000000000 +0100
+++ amule-1.2.6+rc8/src/ListenSocket.cpp	2005-02-15 11:50:39.000000000 +0100
@@ -1988,7 +1988,7 @@
 			} else {
 				strError = wxT("An unnamed client");
 			}
-			strError += wxString::Format(wxT(" (IP:%s) caused a socket blocking error."),unicode2char(m_client->GetFullIP()));
+			strError += wxT(" (IP:") + m_client->GetFullIP() + _(") caused a socket blocking error or closed connection.");
 		} else {
 			strError = wxT("A client caused a socket blocking error.");
 		}

