--- amule-2.1.0/src/OtherFunctions.cpp~	2006-01-01 06:17:25.000000000 +0100
+++ amule-2.1.0/src/OtherFunctions.cpp	2006-01-12 17:56:19.000000000 +0100
@@ -211,7 +211,8 @@
 	if ( ext==wxT("mpc")  || ext==wxT("mp4")  || ext==wxT("aac")  || ext==wxT("ape")  ||
 	     ext==wxT("mp3")  || ext==wxT("mp2")  || ext==wxT("wav")  || ext==wxT("au")   ||
 		 ext==wxT("ogg")  || ext==wxT("wma")  || ext==wxT("rma")  || ext==wxT("mid")  ||
-		 ext==wxT("m4a")  || ext==wxT("m4b")  || ext==wxT("m4p") )
+		 ext==wxT("m4a")  || ext==wxT("m4b")  || ext==wxT("m4p") || ext==wxT("flac") ||
+		 ext==wxT("mka") )
 		 return ftAudio;
 
 	if ( ext==wxT("jpg")  || ext==wxT("jpeg") || ext==wxT("bmp")  || ext==wxT("gif")  ||
